pennylane/devices/default_qubit.py

```diff
@@ -95,6 +114,18 @@ def __init__(self, wires, *, shots=1000, analytic=True):
         self._state = self._create_basis_state(0)
         self._pre_rotated_state = self._state
 
+        self._ops_map = {
```

Comment:
I'm thinking maybe `self._ops_apply_fn` or `self._apply_fns` might be more descriptive here? The `map` is a bit redundant, and doesn't really describe what the mapping _is_.